buyButton/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Buy button</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="style.css" />
+</head>
+<body>
+    <div class='buy-button-wrapper'>
+        <div class='buy-button'>
+            <div class='minus unclickable'>
+                <span class='line line-minus'></span>
+            </div>
+            <div id='count-box'>
+                <p class='less'>0</p>
+                <p class='actual'>1</p>
+                <p class='more'>2</p>
+            </div>
+            <div class='plus'>
+                <span class='line line-plus-1'></span>
+                <span class='line line-plus-2'></span>
+            </div>
+            <div class='total-price'>
+                <svg xmlns="http://www.w3.org/2000/svg" version="1.1" viewbox='0 0 26 28'>
+                    <path d="M10 24c0 1.094-0.906 2-2 2s-2-0.906-2-2 0.906-2 2-2 2 0.906 2 2zM24 24c0 1.094-0.906 2-2 2s-2-0.906-2-2 0.906-2 2-2 2 0.906 2 2zM26 7v8c0 0.5-0.391 0.938-0.891 1l-16.312 1.906c0.078 0.359 0.203 0.719 0.203 1.094 0 0.359-0.219 0.688-0.375 1h14.375c0.547 0 1 0.453 1 1s-0.453 1-1 1h-16c-0.547 0-1-0.453-1-1 0-0.484 0.703-1.656 0.953-2.141l-2.766-12.859h-3.187c-0.547 0-1-0.453-1-1s0.453-1 1-1h4c1.047 0 1.078 1.25 1.234 2h18.766c0.547 0 1 0.453 1 1z"></path>
+                </svg>
+                <p>$35</p>
+            </div>
+        </div>
+    </div>
+    <script src="js.js"></script>
```

Comment:
can you please name this file at list as script.js and add type='text/javascript' ?

Reply:
Done,
Sure i have changed it in all the files :)